tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -1750,46 +1979,141 @@ def test_content_analyst_verify_ap_biology_questions_111274(self):
         """Verify question availability for Biology for AP Courses.
 
         Steps:
+        Log in as Content Analyst
+        Open the drop-down menu
+        Click "QA Contents"
+        Select "Biology For AP® Courses" in
+         the "available-books drop-down list"
+        Click a non-introductory section
 
         Expected Result:
+        Exercises are displayed.
         """
-        self.ps.test_updates['name'] = 't1.58.030' \
+        self.ps.test_updates['name'] = 't1.58.027' \
             + inspect.currentframe().f_code.co_name[4:]
         self.ps.test_updates['tags'] = [
             't1',
             't1.58',
-            't1.58.030',
-            '111274'
+            't1.58.027',
+            '111271'
```

Comment:
watch out for copy/paste errors in coding - mismatch of tags and name; verify all name/tag/id match for a function